pkg/usagetracker/usagetrackerpb/usagetracker.proto

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+
+syntax = "proto3";
+
+package usagetrackerpb;
+
+option go_package = "usagetrackerpb";
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+
+service UsageTracker {
+  rpc TrackSeries(TrackSeriesRequest) returns (TrackSeriesResponse);
+}
+
+message TrackSeriesRequest {
+  // The tenant owning the series.
+  string userID = 1;
+
+  // The hashes of the series to track.
+  repeated int64 seriesHashes = 2;
```

Comment:
Looks good as a first approach, but given the horrible preformance of grpc unmarshaling of slices (it doesn't build the slice of a correct length), we may want to switch to just a []byte and read that from the service implementation in a streamed fashion.

That can be done in a future version though.

Reply:
I agree. Maybe we can keep this for a production version, rather in the prototype. Looks something we know how to optimize.